src/ds/test/map_test.cpp

```diff
@@ -21,6 +21,14 @@ struct CollisionHash
 using K = uint64_t;
 using V = uint64_t;
 
+namespace map
+{
+  inline size_t get_size(uint64_t n)
```

Comment:
What's n?

Reply:
Any `uint64_t`? These are the free functions used by champ/RB maps to work out how big their serialized values are.
```suggestion
  inline size_t get_size(uint64_t)
```